backend/azure-functions/Models/IExample.ts

```diff
@@ -0,0 +1,8 @@
+export interface IExample {
```

Comment:
if i can it right this is a database example entity ?
i suggestion make the "write" db function generic (accept any object) and write any data, not coupled to any model atm

Reply:
to make it easy to extend, 
if you think of the steps to extend the write to database to more entities, with the current implement we'll need to add an interface here, and i noticed a dataservice, maybe more ? it is too much imo, it could end with `soomeDBService.write(objectToWrite: any)` (any could be replace with a very basic model to enforce an `_id`) and we reach from anywhere in the code without creating more models

Reply:
The steps to add more entities really depend on what they and how they are used. I'm just somewhat afraid to work with very general objects, since we basically lose our ability to type-check our entities (and we're working with typescript after all). It will also be harder to keep up what entities do we have, and what properties do they have, since it won't be defined anywhere. It will be really easy to get lost, in my opinion.

Reply:
On the same topic - I would also like to validate the body of incoming requests (can be done with a library called Joi), so that incoming requests are conforming to our entities structure and have valid values. Didn't include it in the example in order to keep the example simple (mentioned it in the comment in the code as well).